css/src/components/notification.scss

```diff
@@ -37,71 +40,70 @@ $notification-dismiss-margin: $spacer-2 !default;
 		&.notification-#{$name} {
 			border-color: $dark;
 			background-color: $background;
-			color: $dark;
 
 			&.is-loading::before {
 				border-color: transparent transparent $dark $dark;
 			}
+
+			.notification-icon {
+				color: $dark;
+			}
 		}
 	}
 
 	&.is-loading {
 		color: transparent;
 
 		> :first-child {
-			margin-inline-start: calc($notification-padding + 0.375em);
+			margin-inline-start: 1.75em;
```

Comment:
The margin here doesn't seem to make a visual impact. Is it needed?

Reply:
it doesn't make a visual impact cause it's visually hidden :D check the devtools. make it visible. This spacing is needed for a case text will become visible so it doesn't overlap with loading icon